src/spreadinterp.cpp

```diff
@@ -263,15 +314,17 @@ int spreadinterpSorted(BIGINT *sort_indices, BIGINT N1, BIGINT N2, BIGINT N3,
 
   else // ================= direction 2 (interpolation) ===========
     interpSorted(sort_indices, N1, N2, N3, data_uniform, M, kx, ky, kz, data_nonuniform,
-                 opts, did_sort);
+                 opts);
```

Comment:
ok, I guess did_sort not used in interp (dir=2) case; this may reappear some day...